creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -76,14 +77,13 @@ class SnapbackSM {
     })
 
     // Config to determine if reconfig is enabled
-    this.snapbackReconfigEnabled = (typeof this.audiusLibs.snapbackDevModeEnabled === 'boolean' && this.audiusLibs.snapbackDevModeEnabled) ||
-      this.nodeConfig.get('snapbackReconfigEnabled')
+    this.snapbackReconfigEnabled = this.nodeConfig.get('snapbackReconfigEnabled')
```

Comment:
wait i thought we were gonna keep the `snapbackDevModeEnabled` check here as well?

Reply:
I think this flag should be independent of `snapbackDevModeEnabled` since in local/prod mode, we want reconfig on/off depending on how we configure it

Reply:
nvm, makes sense